test/test_transforms.py

```diff
@@ -180,6 +180,14 @@ def test_randomperspective(self):
                                torch.nn.functional.mse_loss(tr_img2, F.to_tensor(img)))
 
     def test_randomperspective_fill(self):
+
+        # assert fill being either a Sequence or a Number
+        with self.assertRaises(TypeError):
+            transforms.RandomPerspective(fill=NotImplemented)
```

Comment:
nit: let's replace `NotImplemented` by `{}` or something else. Seems like NotImplemented has a special meaning for python: https://docs.python.org/3/library/constants.html#NotImplemented